CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - **Typings**
   - `Window` interface with custom methods.
 
+- **`EditorProvider`**
+  - Add `emitLocaleEventToIframe` method to sync iframe locale.
```

Comment:
:pick: No need to use the word `Add` here, since it's already under the `Added` section.